docs/src/conf.py

```diff
@@ -191,6 +191,15 @@ def autolog(message: str, section: str | None = None, color: str | None = None)
 tippy_enable_doitips = False
 tippy_rtd_urls = [
     "https://www.sphinx-doc.org/en/master/",
+    "https://geovista.readthedocs.io/en/latest/",
+    "https://geovista.readthedocs.io/en/stable/",
+    "http://localhost:11000",
+    "http://0.0.0.0:11000",
```

Comment:
Including these for developers building the docs and serving them locally, which allows `sphinx-tippy` to render local tooltip references.